src/components/HeaderUSWDSSelections.vue

```diff
@@ -6,9 +6,9 @@
         <div class="usa-navbar">
           <div class="usa-logo" id="basic-logo">
             <em class="usa-logo__text"
-              ><a href="/" title="Home" aria-label="Home">{{
-                titleForSelectionHeader
-              }}</a></em
+              ><h4 href="/" title="Home" aria-label="Home">
```

Comment:
I recall Mary was recommending the `h1` tag for this.

Reply:
We tried out the h1 tag but we felt the text was unnecessarily large in the non mobile view.  We thought h4 was a better fit.  This is what H1 looks like.
![h1](https://user-images.githubusercontent.com/53274375/62639328-ebe26000-b904-11e9-9cdf-d1420bbe8c9e.PNG)

Reply:
Hmm... that's kinda ugly. Is there a way to prevent it from wrapping?

Reply:
I will look into it